.github/workflows/trigger_prerelease.yaml

```diff
@@ -83,4 +108,4 @@ jobs:
         with:
           slack-bot-user-oauth-access-token: ${{ secrets.slack_token }}
           slack-channel: ${{ secrets.slack_channel }}
-          slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
+          slack-text: "❌ `${{ github.event.repository.full_name }}`: [prerelease trigger failed](${{ github.server_url }}/${{ github.event.repository.full_name }}/actions/runs/${{ github.run_id }})"
```

Comment:
The link was tested using [this testing workflow](https://github.com/sigilioso/settings-test/actions/runs/5123398858/jobs/9213851524) (temporal link as it is an action-run)